src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/IDictionaryExtensions.cs

```diff
@@ -64,6 +64,25 @@ internal static class IDictionaryExtensions
             builder.Add(value);
         }
 
+        public static bool MultiAdd<TKey, TValue>(this IDictionary<TKey, ImmutableHashSet<TValue>> dictionary, TKey key, TValue value, IEqualityComparer<TValue>? comparer = null)
+            where TKey : notnull
```

Comment:
did any callers pass the comparer?  if not, consider removing.

Reply:
I like to keep these available in general-purpose helpers, especially since the change wouldn't save any logic for this method.